portfolio-project-matching-app/pages/project.js

```diff
@@ -0,0 +1,88 @@
+import { useRouter } from 'next/router';
+import { useEffect, useState } from 'react';
+import { getProjectById } from '../backend/dao';
+import Button from '../components/Button'
+import UserIcon from '../components/UserIcon';
+
+const Project = () => {
+    const router = useRouter();
+    console.log(router.query);
+    const [project, setProject] = useState();
+
```

Comment:
Nice, I like the utilization of the `query` object from the `useRouter` hook! Also good use of the backend functions Logan created within the `useEffect` block.